src/components/ErrorInfo/ErrorInfo.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { ErrorInfoProps } from './ErrorInfoProps';
+
+export const ErrorInfo: FC<ErrorInfoProps> = ({
+  visibleError,
+  setError,
+}) => {
+  const removeError = () => {
+    setError('');
+  };
+
+  setTimeout(() => {
+    if (visibleError) {
+      removeError();
+    }
+  }, 3000);
+
+  return (
+    <div className={cn(
+      'notification', 'is-danger', 'is-light', 'has-text-weight-normal',
+      {
+        hidden: !visibleError,
+      },
+    )}
+    >
+      <button
+        aria-label="remove-error"
+        type="button"
+        className="delete"
+        onClick={removeError}
+      />
+
+      {visibleError}
+
```

Comment:
Do not forget about good formatting (do not leave blank lines if it is not necessary).